Cargo.toml

```diff
@@ -74,6 +75,7 @@ lightbeam = [
     "wasmtime-wast/lightbeam",
     "wasmtime/lightbeam",
 ]
+profiling = ["wasmtime-jit/profiling", "wasmtime/profiling", "wasmtime-profiling/profiling"]
```

Comment:
Hi @yurydelendik. Thanks for the comment. So you are thinking no feature at all? In a previous comment: https://github.com/bytecodealliance/wasmtime/pull/360#discussion_r328848191
it was suggested to use a feature to avoid bringing in crates used by jitdump.  This feature add was to accomplish that. It would also hide logic for https://github.com/bytecodealliance/wasmtime/pull/819.

Reply:
I definitely wouldn't want to have those extra dependencies pulled in unconditionally for all wasmtime users.

Reply:
I'll go ahead and push a new update to test, but based on your confirming the removal of the profiling feature I'll just make the change and resubmit.

Reply:
I see. It was not clear to me first that it prefs the jitdump -- we can keep it. Can we perhaps rename this feature?

Reply:
@yurydelendik Sure .. open to suggestions?

Reply:
In "wasmtime-profiling/Cargo.toml", the "jitdump" feature name shows better what is enabled. Similarly "jitdump" can be used in wastime-jit or this Cargo.toml, for name -- we can give better/group name later as we land more profilers support.

Reply:
Should be done.